Standards/CCSMB-11.md

```diff
@@ -0,0 +1,107 @@
+
+Software MAY use this variable when determining what graphical OS or desktop manager is running.
+
+Compliant graphical OSes and desktop managers SHOULD set this variable to `nil` when exiting.
+
+## Module
+
+Compliant graphical OSes and desktop managers MAY define the `ccsmb_desktop` module. The module MAY be loaded through `require`; it MAY also be defined in the `ccsmb_desktop` global, but this is strongly discouraged.
+
+The `ccsmb_desktop` module MUST contain the `ccsmb_desktop.run` function. The function MUST open a new window. The function MUST accept arguments in the same way that `shell.run` does:
+> All arguments are concatenated together and then parsed as a command line. As a result, shell.run("program a b") is the same as shell.run("program", "a", "b").
+> — [CC: Tweaked documentation](https://tweaked.cc/module/shell.html#v:run)
+
+The function MAY fail if the application fails to launch or continue running for any reason.
+
+The `ccsmb_desktop` module MUST NOT contain any functions not specified above.
+
+## Settings
+
+This section defines variables that SHOULD be defined through the `settings` API ("settings").
+
+### `ccsmb.desktop.search_targets`
+
+This setting MUST be an array of absolute directories. Each directory SHOULD contain valid application specifications.
+
+Compliant software MUST search all directories (in the order listed). Compliant software MUST NOT assume the filenames in each directory are the same as the application names; however, software management tooling SHOULD install application specifications with a sensible filename. Application specifications SHOULD use the extension `.desktop`; `.lua` SHOULD also be allowed, but software management tooling SHOULD avoid installing specifications with this extension, to avoid confusion with actual programs.
+
+If this setting is not set, compliant software MAY search in the following directories in the order listed:
+1. `/Application Specifications`
+2. `/AppSpecs`
+3. `/etc/ccsmb/applications`
+4. `/usr/share/ccsmb/applications`
+5. `/Applications`
+6. `/etc/applications`
+7. `/usr/share/applications`
+8. `/Library/Application Specifications`
+
+If none of the directories listed above exist, compliant software MAY create a directory of its choice, provided it sets this setting and `ccsmb.desktop.install_target` (defined below).
+
+If none of the directories listed aboove exist, application launchers MAY choose to abort launching themselves or to remain open and empty.
+
+### `ccsmb.desktop.install_target`
+
+This setting MUST be a string containing a path to a directory. The directory MUST be created. Compliant software management tooling MUST install applications in this directory.
```

Comment:
maybe you should mention that the path must start with a `/` and must not end with a `/`, the same way as in ccsmb-10